cppsrc/linux/sound-mixer.cpp

```diff
@@ -210,6 +210,7 @@ Napi::Value AudioSessionObject::New(Napi::Env env, void *data)
     Napi::ObjectWrap<AudioSessionObject>::Unwrap(result)->pSession = session;
     result.Set("name", session->description());
     result.Set("appName", session->appName());
+    result.Set("state", 1);
```

Comment:
This is overwritten by the getter you added, this line is useless.

Reply:
Are you sure ? I did not add the getter to the linux/ file and instead left it as this because it does not need to change. I only added the getter to the win/ implementation.

Reply:
Oh yeah I did not see this. In this case, I guess it is okay.